Lab2.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lab 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sierra Wilde"
```

Comment:
Olivia asked us to put both our names here so it is easier for her to figure out partners and grade